pkg/api/core/seed/warnings.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package seed
+
+import (
+	"context"
+
+	"github.com/gardener/gardener/pkg/apis/core"
+)
+
+// GetWarnings returns warnings for the provided seed.
+func GetWarnings(_ context.Context, seed, _ *core.Seed) []string {
```

Comment:
These warnings don't make a lot of sense in the light that `endpoint`/`weeder` will be kept in sync. They will always be shown even if users already switched. I would drop them.